client/src/containers/SingleArticle/SingleArticle.js

```diff
@@ -23,7 +23,7 @@ class SingleArticle extends Component {
   showArticleImage() {
     if (this.state.article.articleImage) {
       return (<div className="writer-picutre">
-        <img src={`http://localhost:3001/images/${this.state.article.articleImage}.jpg`} alt={this.state.article.articleImage} />
+        <img src={`http://localhost:3001/images/${this.state.article.articleImage}`} alt={this.state.article.articleImage} />
```

Comment:
it should not be `localhost:3000` otherwise it will not work on heroku .. use a similar technique to what you did on apiClient to get the server name (or default to localhost)